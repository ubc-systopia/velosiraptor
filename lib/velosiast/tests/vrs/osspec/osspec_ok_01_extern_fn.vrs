const FOO : int = 1234;

osspec MyOs {
    const BAR : int = 1234;

    extern type Capref {}

    extern fn vnode_map(src: Capref, dst: Capref, offset: int) -> bool
}

