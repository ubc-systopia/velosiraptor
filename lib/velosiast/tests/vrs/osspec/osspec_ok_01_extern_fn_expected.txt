VelosiAst(tests/vrs/osspec/osspec_ok_01_extern_fn.vrs)
--------------------------------------------------------

const FOO : int = 0x4d2;

osspec MyOs () {

  const BAR : int = 0x4d2;

  extern type Capref

  extern fn vnode_map(src: Capref, dst: Capref, offset: int) -> bool
    requires true;

}

--------------------------------------------------------
